Scripts/Bots/Twitter_bot/Twitter_bot.py

```diff
@@ -0,0 +1,21 @@
+from twython import Twython
+from twython import TwythonStreamer
+class MyStreamer(TwythonStreamer):
+	def on_success(self, data):
+		if 'text' in data:
+			a=data['text'].lower()
+			username=data['user']['screen_name']
+			id=data['id']
+			st.create_favorite(id=id)
+   			#change 'nice tweet' too your desired retweet reply
+			st.update_status(status='Nice Tweet @'+username, in_reply_to_status_id=id)
+#enter your unique keys and tokens
+api_k='*'               
```

Comment:
please enter your unique keys and tokens in config files and parse it utilize it in your code

Reply:
But I get a limit of around 100 requests, how can I enter my keys and tokens in public code, the user should generate it on their own.

Reply:
just add token  code in separate file (don't add your token :smile:   )

Reply:
Are you trying to say a code to generate tokens for api?